track.c

```diff
@@ -5,7 +5,7 @@
  *    Copyright 2000, Jeroen Vreeken
  *    This program is published under the GNU Public license
  */
-
+#include <linux/videodev2.h>
```

Comment:
So this is a known issue, however it is not motion's fault, but rather OS (only some affected). I don't think it is worth changing the code just to workaround that issue. Check this thread: https://github.com/Mr-Dave/motion/pull/71